devices/bosch/room_thermostat2.json

```diff
@@ -28,7 +28,7 @@
       },
       "meta": {
         "values": {
-          "config/mode": {"auto": 0, "heat": 1, "off": 5}
+          "config/mode": {"auto": 0, "manual": 1, "off": 5}
```

Comment:
Please do not change the names here. Those are aligned with the respective zigbee attribute and REST API clients do not expect different values. We should stay consistent here.

Reply:
Why? I don't understand. Here are two screenshots of unchanged devices.

Eurotronic SPZB0001:
![Eurotronic_SPZB0001](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/80219712/44a1a599-5e68-4b44-899c-d82c44a8ddc5)

Bosch radiator thermostat II:
![Bosch_radiator_thermostat_II](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/80219712/4d8ed3f8-0209-493c-853c-9c108b5b6bfe)

Reply:
How about if I use mode/preset instead of devicemode/mode?

mode --> heat, cool and off
preset --> manual, auto and holiday